### PR TITLE
Fix erlcloud_sns:publish content format

### PR DESCRIPTION
**Pull request by aleklisi**
This PR fixes issue with setting `MessageStructure` option properly (to `json`) in   `erlcloud_sns:publish/5`.

### CODE REVIEW

**Comment by mongoose-im** [comment]
[6047.1](https://travis-ci.org/esl/MongooseIM/jobs/474329999) / Erlang 19.3 / small_tests / b6d8c969be649a114c1163556c9a1d6cf759d482
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2176/6047/small_tests.19.3) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2176/6047/small_tests.19.3/small/ct_run.mongooseim@localhost.2019-01-02_09.26.53/lib.mongooseim.logs/run.2019-01-02_09.26.54/suite.log.html)

File: src/event_pusher/mod_event_pusher_sns.erl [hunk 1]
```diff
@@ -153,8 +153,8 @@ try_publish(Host, TopicARN, Content, Attributes, Retry) ->
 -spec publish(Host :: jid:lserver(), topic_arn(), Content :: jiffy:json_value(),
               attributes()) -> MessageId :: string().
 publish(Host, TopicARN, Content, Attributes) ->
-    EncodedContent = jiffy:encode(Content),
-    erlcloud_sns:publish(topic, TopicARN, unicode:characters_to_list(EncodedContent),
+    EncodedContent = maps:to_list(Content),
```
[the remark this answers is not in the document]
Comment (aleklisi) [reply src/event_pusher/mod_event_pusher_sns.erl]: kk

**Comment by mongoose-im** [comment]
[6050.1](https://travis-ci.org/esl/MongooseIM/jobs/474407312) / Erlang 19.3 / small_tests / 957243606effb9374c0c46d9f273e58fecb9b7c6
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2176/6050/small_tests.19.3) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2176/6050/small_tests.19.3/small/ct_run.mongooseim@localhost.2019-01-02_13.48.40/lib.mongooseim.logs/run.2019-01-02_13.48.41/suite.log.html)

**Comment by mongoose-im** [comment]
[6053.1](https://travis-ci.org/esl/MongooseIM/jobs/474429107) / Erlang 19.3 / small_tests / d6a7325a1a9dc9d447455d893d99f259bc3a9a41
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2176/6053/small_tests.19.3) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2176/6053/small_tests.19.3/small/ct_run.mongooseim@localhost.2019-01-02_14.45.41/lib.mongooseim.logs/run.2019-01-02_14.45.42/suite.log.html)

**Comment by mongoose-im** [comment]
[6055.1](https://travis-ci.org/esl/MongooseIM/jobs/474746083) / Erlang 19.3 / small_tests / f4602237b311e625aca646179757eb4186986c2d
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2176/6055/small_tests.19.3) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2176/6055/small_tests.19.3/small/ct_run.mongooseim@localhost.2019-01-03_08.10.48/lib.mongooseim.logs/run.2019-01-03_08.10.49/suite.log.html)

**Pull request merged**